yo1k/sandbox/postgresql/pgexercises/readme.md

```diff
@@ -0,0 +1,11 @@
+Contains some solutions to programming exercises 
+from [pgexercises.com](https://pgexercises.com/gettingstarted.html)
+using [Psycopg](https://www.psycopg.org/docs/index.html) database adapter for the Python.
+
+It includes `clubdata.sql` script for creation a database.
```

Comment:
creation -> creating
a database -> the database used in the exercises

Reply:
Done.